.env

```
@@ -93,7 +93,7 @@ DEVTOOLSET_VERSION=-1
 # Please also update the crossbow configuration in order to keep the github
 # actions cache up to date for the macOS wheels:
 #   https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml
-VCPKG="38bb87c"
+VCPKG="7ef97003d78178948068ccd6fa7964f938f41c76"
```

Please use short SHA as the above comment says:

```suggestion
VCPKG="7ef9700"
```